lumen_runtime/src/time/datetime.rs

```diff
@@ -0,0 +1,13 @@
+cfg_if::cfg_if! {
+  if #[cfg(all(target_arch = "wasm32", feature = "time_web_sys"))] {
+     mod web_sys;
+     pub use self::web_sys::*;
+  } else {
+     mod std;
+     pub use self::std::*;
+  }
+}
+
+pub fn utc_now() -> [usize; 6] {
+    return get_utc_now();
```

Comment:
You don't need an explicit `return` if you don't use the `;` at the end of the last statement in a function.  In Rust `;` makes something a statement and then its return is `()`, pronounce "unit" and without `;`, it is an expression like how it works in Ruby and Elixir.